doc/recognition.xml

```diff
@@ -416,6 +419,19 @@ elements that are generated from them remember, how they were acquired.
 </Description>
 </ManSection>
 
+<ManSection>
+<Attr Name="HomomSourceCheck" Arg="ri"/>
+<Description>
+    The value of this attribute is a function which returns a boolean
+    which is <K>false</K> for values which are not in the source of
```

Comment:
What does "values" refer to? What values? Perhaps the function returned is supposed to be unary (?), and "values" returns to any value passed to it as an argument?

Reply:
Yes, the first 'value' is the function, the second 'value' is arguments to this function (the function is unary)